ansible_collections/arista/avd/roles/eos_validate_state/python_modules/tests/avdtestconnectivity.py

```diff
@@ -13,6 +13,35 @@
 LOGGER = logging.getLogger(__name__)
 
 
+class AvdTestBaseReachability(AvdTestBase):
```

Comment:
Let's not use a base class but instead move `is_vtep` and `is_wan_vtep` to `DeviceUtilsMixin`. Also, I agree that creating a function to avoid duplicate codes to create the tests is cleaner but for now let's keep the existing code as it is to faciliate the migration to PyAVD.

Reply:
updated